content/docs/buildpack-author-guide/caching-strategies.md

```diff
@@ -48,7 +35,7 @@ This will guarantee that the previously published application image layer in the
 
 Setting `build = true` makes a layer available to subsequent buildpacks.  Therefore binaries installed to the `bin` directory on a `build = true` layer are available to subsequent buildpacks during the build phase.  It is also the case that `lib` directories on a `build = true` later are added to the `LD_LIBRARY_PATH` during the build phase of subsequent buildpacks.  Environment variables defined in a `build = true` layer are similarly available.  For any layer where `launch = true` and `build = true`, a launch layer from the OCI registry can no longer be reused. Instead, the layer must be made available locally so that subsequent buildpacks can use it.
 
-Setting `cache = true` allows additional fine-grained control over caching.  The `cache = true` flag caches a layer and allows a buildpack to make _content_ level decisions about the validity of the cache (as opposed to using the less granular metadata).  As an example, suppose a layer where `launch = true` installs a `jq` binary with version `1.5` and sets `version=1.5` in the layer metadata.  By default, this layer will not be re-used from the registry when a buildpack requests `jq` with `version=1.6` to be installed.  However, setting `cache = true` makes a previously built layer available during the build.  A buildpack could then prefer to implement logic to restore `jq` with `version=1.5` instead of performing a potentially expensive download of `jq` with `version=1.6`.  The `cache = true` setting allows for cache validation decisions to be made at a level of granularity that is much finer grained than layer metadata.
+Setting `cache = true` keeps a local cache of the layer.  The cache is restored before the buildpacks build phase.  
```

Comment:
The cache may or may not be local. Cached layers can also be stored in the registry as a cache image.

Reply:
Setting `cache = true` implies that the layer is restored locally before the buildpacks build phase.